rust_src/src/decompress.rs

```diff
@@ -96,7 +102,15 @@ pub fn zlib_decompress_region(mut start: LispObject, mut end: LispObject) -> boo
             // Decompress all data finished.
             Ok(0) => {
                 // Delete the compressed data.
-                unsafe { del_range(istart, iend) };
+                unsafe {
+                    del_range_2(
+                        istart, istart, /* byte, char offsets the same */
```

Comment:
```suggestion
                        istart, istart, // byte, char offsets the same
```

We usually use `//` style comments in Remacs.

Reply:
Yeah, you are right.

Reply:
Fixed.